src/main/java/cr0s/warpdrive/conf/WarpDriveConfig.java

```diff
@@ -759,13 +793,21 @@ private static void loadAdvancedRepulsionSystems() {
 
 	public static Block getDefaultSurfaceBlock(Random random, boolean corrupted, boolean isMoon) {
 		if (isMoon) {
-			if (random.nextInt(5) == 1) {
+			if (isIndustrialCraft2loaded && random.nextInt(10) == 1)
```

Comment:
These world gen changes will be deleted along with this form of world gen soon, but they don't hurt anything for the present